griptape/drivers/vector/pgvector_vector_store_driver.py

```diff
@@ -52,11 +53,11 @@ def validate_engine(self, _, engine: Optional[Engine]) -> None:
             raise ValueError("An engine or connection string is required")
 
     def __attrs_post_init__(self) -> None:
-        """If a an engine is provided, it will be used to connect to the database.
+        """If an engine is provided, it will be used to connect to the database.
         If not, a connection string is used to create a new database connection here.
         """
         if self.engine is None:
-            self.engine = create_engine(self.connection_string, **self.create_engine_params)
+            self.engine = create_engine(self.connection_string, **self.create_engine_params)  # pyright: ignore
```

Comment:
What was the error that prompted the ignore?

Reply:
`Cannot assign member "engine" for type "PgVectorVectorStoreDriver*" ` which seems to point to a problem with SQL Alchemy

Reply:
Let's try this:
`from typing import cast`

`self.engine = cast(Engine, create_engine(self.connection_string, **self.create_engine_params))`